src/pipeline_manager.py

```diff
@@ -87,15 +88,35 @@ def train(pipeline_name, dev_mode, logger, params, seed):
     meta_train = meta[meta['is_train'] == 1]
     meta_valid = meta[meta['is_valid'] == 1]
 
+    train_mode = True
+
     meta_valid = meta_valid.sample(int(params.evaluation_data_sample), random_state=seed)
 
     if dev_mode:
         meta_train = meta_train.sample(20, random_state=seed)
         meta_valid = meta_valid.sample(10, random_state=seed)
 
+    if 'lgbm' in pipeline_name:
```

Comment:
@apyskir since this is stircly for the lgbm_train I would explicitly go if `piepline_name==lgbm_train`:

Reply:
@jakubczakon Currently it's strictly for `pipeline_name=='lgbm', actually. OK.

Reply:
@jakubczakon ...and now it's `pipeline_name=='scoring_model'`